### DISCUSSION TITLE
Apache NetBeans 16 release candidates

### DISCUSSION THREAD

**Discussion by neilcsmith-net** [apache/netbeans#4815] [General]
Release candidates for the next release of Apache NetBeans will be available weekly until release from -

https://nightlies.apache.org/netbeans/candidate/netbeans/

Your help in testing, reporting and verifying issues would be highly appreciated by the whole Apache NetBeans community.

IMPORTANT : These release candidates are pre-release software, and not official releases of Apache NetBeans. Only binary and source zips are available at the above link.  Other artefacts, such as unsigned installers, VSCode plugin, etc. are linked from announcements on the [dev@ mailing list](https://lists.apache.org/list.html?dev@netbeans.apache.org).

**Comment by wumpz** [comment]
Since a couple of days, Netbeans 16 was released: https://github.com/apache/netbeans/releases/tag/16. So when is the installer at https://netbeans.apache.org/download/index.html available?

**Reply by mbien** [reply]
Hi @wumpz, 
NB 16 isn't officially released yet. Some votes are still in progress - but it should be soon.

@ebarboni @neilcsmith-net can we keep the release notes as draft next round until the actual release, since users might be subscribing to the github releases and get the notification too early. We should also add a link, linking to the download page combined with a note that the sources github generates are not the actual apache artifacts.

**Reply by wumpz** [reply]
A sorry. I thought since Github presented it as if it already has been released.

**Reply by ebarboni** [reply]
Sorry for anticipation, the main vote passes but no notification or web site changes until all installer stack/vs code is ready.

I look at a lot of other "Apache project" the release artefacts are let with no notice, no download.

**Reply by neilcsmith-net** [reply]
Yes, NetBeans 16 was officially released on Nov 30th - https://lists.apache.org/thread/oo6bmxtorb4rtn2v2dxqbq1hgp8tx4x6  The binary zip was made available that day from https://downloads.apache.org/netbeans/netbeans/  The community installers were made available the day after - https://www.codelerity.com/netbeans/  Some ASF installers are already released at https://downloads.apache.org/netbeans/netbeans-installers/

Unfortunately this process takes about a week, and the website update is the last step.  It would be good if we could speed this up a little.

Binaries will always show up at https://downloads.apache.org/netbeans/ immediately they're released, even if the NetBeans website is a little behind.

**Reply by mbien** [reply]
i mean... yes the main artifacts are released. All I am saying is that we probably should not call it released until users can download it from the website (the zip at least). Since this can create many disappointed users when they all get the notification via github but there is nothing to download (unless you look at the mailing list).

**Reply by neilcsmith-net** [reply]
@mbien that applies to a whole bunch of channels, not just GH notification.  The only real way to handle that is to close all votes concurrently and move everything into place at the same time.  Pros and cons.  We could still do with an extra bod on the release team, and there's still a seat with your name on it! :wink:

**Reply by mbien** [reply]
download page just got updated:
https://netbeans.apache.org/download/
